src/main.rs

```diff
@@ -30,6 +30,8 @@ async fn main() {
     bot.add_command(commands::makersuite::GoogleGemini);
     bot.add_command(commands::makersuite::GooglePalm);
     bot.add_command(commands::groq::Llama);
+    bot.add_command(commands::fal::Fal::realistic_vision());
+    bot.add_command(commands::fal::Fal::sdxl_lightning());
```

Comment:
add these after `CraiyonSearch`

Reply:
may i ask why?

Reply:
so that all image generation commands are grouped together. you can also place them after stable horde commands